web-ui/src/components/member-directory/MemberSummaryCard.jsx

```diff
@@ -12,17 +14,33 @@ import SplitButton from "../split-button/SplitButton";
 
 const MemberSummaryCard = ({ member, index }) => {
   const { state, dispatch } = useContext(AppContext);
-  const { memberProfiles, userProfile } = state;
+  const { csrf, memberProfiles, userProfile } = state;
   const isAdmin =
     userProfile && userProfile.role && userProfile.role.includes("ADMIN");
-  const { location, name, workEmail, title } = member;
+  const { location, name, workEmail, title, supervisorid } = member;
   const [currentMember, setCurrentMember] = useState(member);
-  const [open, setOpen] = useState(false);
+  const [supervisorName, setSupervisorName] = useState();
 
+  const [open, setOpen] = useState(false);
   const handleOpen = () => setOpen(true);
 
   const handleClose = () => setOpen(false);
 
+  // Get Supervisor's name
+  useEffect(() => {
```

Comment:
Are we worried about doing an extra request for each record slowing down page load? This being the member directory page we already have every member loaded into client memory or the supervisor name can be made a field on the response DTO.

Reply:
That's a good point. I was having trouble accessing the member profile on that page, the call was a fallback.  Let me take another look at that.